src/org/usfirst/frc/team694/robot/commands/auton/DrivetrainMoveToLineCommand.java

```diff
@@ -10,10 +10,10 @@
 public class DrivetrainMoveToLineCommand extends Command {
     private boolean isReached = false; 
     private double distance = 0;
-    private double speedMode = 0;
+    private double speedNumber = 0;// 0,1, or 2, repesents speeds 0.25,0.5,0.75 respectively.
```

Comment:
Do not use this. You're literally converting a speed into an imaginary "speed number" unit and then converting it back to a speed when you use it, which makes it pointless. Just use the raw speed value, don't convert it or anything.